ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/Badges/PostLevelByBadgeIdTests.cs

```diff
@@ -0,0 +1,181 @@
+﻿using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models.DTOs;
+using ferrilata_devilline.Services.Interfaces;
+using Newtonsoft.Json;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios.ApiControllers.Badges
+{
+    [Collection("BaseCollection")]
+    public class PostLevelByBadgeIdTests
+    {
+        private readonly TestContext _testContext;
+        private readonly ITokenService _tokenService;
+        
+        private string token;
+
+        public PostLevelByBadgeIdTests(TestContext testContext)
+        {
+            _testContext = testContext;
+            _tokenService = _testContext.TokenService;
+            token = "Bearer " + _tokenService.GenerateToken("useremail@ferillata.com", true);
+        }
+
+        [Fact]
+        public async Task PostLevelById_CorrectAuthentication_CorrectBody_ShouldReturnCreated()
+        {
+            string PostingJson = JsonConvert.SerializeObject(createCorrectLevelInDTO());
```

Comment:
local variables should start with lowercase

Reply:
done